firebase-firestore/src/main/java/com/google/firebase/firestore/local/RemoteDocumentCache.java

```diff
@@ -65,6 +65,20 @@
    */
   Map<DocumentKey, MutableDocument> getAll(Iterable<DocumentKey> documentKeys);
 
+  /**
+   * Looks up the next {@code count} documents for a collection group based on the provided offset.
+   * The ordering is based on the document's read time and key.
+   *
+   * <p>This method may return more results than requested if a collection group scan scans more
+   * than 100 collections.
+   *
+   * @param collectionGroup The collection group to scan.
+   * @param offset The offset to start the scan at.
+   * @param count The number of results to return.
+   * @return A map with next set of documents.
+   */
+  Map<DocumentKey, MutableDocument> getAll(String collectionGroup, IndexOffset offset, int count);
```

Comment:
I kind like the naming scheme since we will have three different `getAll()` methods once https://github.com/firebase/firebase-android-sdk/pull/3201/files lands.

````
Map<DocumentKey, MutableDocument> getAll(Iterable<DocumentKey> documentKeys);
Map<DocumentKey, MutableDocument> getAll(String collectionGroup, IndexOffset offset, int count);
Map<DocumentKey, MutableDocument> getAll(ResourcePath collection, IndexOffset offset);
```

I am going to rename count to limit though.